content/browser/sources.gni

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+brave_content_browser_sources = [
+  "//brave/content/browser/ephemeral_storage_partition.cc",
```

Comment:
since we ended up with adding new files to `content/browser`, is it possible to move code from `browser_context.cc/h` override to a new file? AFAIU that code is not really related to `BrowserContext`

Reply:
E.g. `GetSessionStorageNamespaceId()`